third_party/s2n-bignum/s2n-bignum-imported/arm/aes/aes-xts-enc.S

```diff
@@ -0,0 +1,607 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Pretending these files are imported from s2n-bignum is slightly problematic. The import workflow first asks to delete `s2n-bignum-imported` and then do a clean import. If they don't exist upstream, they will be gone. Can you put them somewhere else?